src/Command/Pull/PullCommandBase.php

```diff
@@ -651,7 +651,14 @@ protected function rsyncFilesFromCloud($chosen_environment, Closure $output_call
     $this->localMachineHelper->getFilesystem()->mkdir($destination);
     $command = [
       'rsync',
-      '-rltDvPhe',
+      // -a archive mode; same as -rlptgoD.
+      // -z compress file data during the transfer.
+      // -v increase verbosity.
+      // -P show progress during transfer.
+      // -k transform symlink to a dir into referent dir.
+      // -h output numbers in a human-readable format.
+      // -e specify the remote shell to use.
+      '-avPhekz',
```

Comment:
Since the `-e` option takes an argument (the shell to use, i.e. ssh), doesn't it need to be the last option in the series?

Reply:
yeeeaaah.